netstandard/ref/netstandard.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <AssemblyVersion>2.0.0.0</AssemblyVersion>
+    <AssemblyVersion>2.1.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Should this be netstandard2.1?

Reply:
You may also want to look into here: https://github.com/dotnet/standard/blob/df61ba30a73c720253c99c5ef592c97f47e74523/netstandard/pkg/NETStandard.Library.pkgproj#L13  As well as what you want the behavior to be when that package is installed in older netstandard versions.  Previously it tried to support the older TFMs but that might not be so scalable now that the package contains the binaries.  It also may not be a requirement anymore since we don't persist the package as a dependency.  Whatever you decide would need to be coordinated with the SDK folks.